bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -1,7 +1,9 @@
 # Generated via https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/query_scheduling/generate_airflow_dags.py
 
 from airflow import DAG
-from operators.task_sensor import ExternalTaskCompletedSensor
```

Comment:
Using `ExternalTaskSensor` instead. This is causing almost all of the DAGs to have some changes. If this is annoying for reviewing this, then I can also open a separate PR for this change.